modules/virt-configuring-aaq-operator.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-understanding-aaq-operator.adoc                
+
+:_mod-docs-content-type: PROCEDURE                                    
+[id="virt-configuring-aaq-operator_{context}"]                                   
+= Configuring the AAQ Operator by using the CLI
+
+You can configure the AAQ Operator by specifying the fields of the `spec.applicationAwareConfig` object in the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+* Update the `HyperConverged` CR by running the following command:
++
+[source,terminal]
+----
+$ oc patch hco kubevirt-hyperconverged -n openshift-cnv --type merge -p '{
+  "spec": {
+    "applicationAwareConfig": {
+      "vmiCalcConfigName": "DedicatedVirtualResources",
+      "namespaceSelector": {
+        "matchLabels": {
+          "app": "my-app"
+        }
+      },
+      "allowApplicationAwareClusterResourceQuota": true
+    }
+  }
+}'
+----
++
+where:
+
+
+`vmiCalcConfigName`:: Specifies how resource counting is managed for pods that run virtual machine (VM) workloads. Possible values are:
++
+--
+* `VmiPodUsage` - Counts compute resources for pods associated with VMs in the same way as native resource quotas and excludes migration-related resources.
```

Comment:
Consider colons rather than dashes here.

Reply:
ISG asks that lists of sentence fragments skip terminating punctuation. 

(For translation's sake, I'd rewrite these as complete sentences and keep the punctuation.)

Reply:
Will replace dashes with colons but I think the sentence formatting is in alignment with the examples in the [contribution guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks).